### PR TITLE
Fix 748

### PR DESCRIPTION
**Pull request by molant**
## Pull request checklist

Make sure you:

- [ ] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [ ] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/contributing/pull-requests/#commitmessages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

## Short description of the change(s)

* Fixes a problem with JSDOM accessing attributes
* Fixes `scripts` in a few rules

### CODE REVIEW

File: scripts/release.ts [hunk 1]
```diff
@@ -292,55 +292,55 @@ const updateSnykSnapshotJSONFile = async () => {
...
-     */
-    await exec(`Create '${SHRINKWRAP_FILE}' `, 'npm shrinkwrap');
-
-    // Publish on `npm`.
-    await exec('Publish on `npm`.', 'npm publish');
-
-    // Restore the package lock file and delete shrinkwrap
-    shell.rm(SHRINKWRAP_FILE);
-    await exec(`Restore ${PACKAGE_LOCK_FILE}`, `git checkout ${PACKAGE_LOCK_FILE}`);
+    // const commitSHAsSinceLastRelease = await getCommitsSinceLastRelease();
```
Comment (molant) [inline scripts/release.ts]: @alrra, no idea why this code is commented. Should I uncomment it?

Comment (alrra) [reply scripts/release.ts]: I'll take care of it.

**Comment by Zirro** [comment]
Note that the attribute issue should be resolved if you upgrade to a more recent release of jsdom. `NamedNodeMap` (used for el.attributes) got several fixes back in 11.4.0.

**Comment by molant** [comment]
@Zirro thanks for the update. I thought we were using the latest version but looks like greenkeeper failed us.

**Pull request closed without merging**